qcodes/instrument_drivers/tektronix/Keithley_s46.py

```diff
@@ -106,6 +106,14 @@ def _set(self, new_state) -> None:
                                "that is not attached to an instrument.")
         self._instrument.write(f":{new_state} (@{self._channel_number})")
 
+    def is_closed(self) -> bool:
+        """
+        A public method to query the channel state without relying on string comparison.
```

Comment:
This part of the docstring seems irrelevant, the description of the return values seems clear enough.
```suggestion
```

Reply:
Ok, I will change this